07-routing-gossip.md

```diff
@@ -270,6 +270,32 @@ nodes not associated with an already known channel are ignored.
    * [`32*byte`:`alias`]
    * [`u16`:`addrlen`]
    * [`addrlen*byte`:`addresses`]
+   * [`node_announcement_tlvs`:`tlvs`]
+
+1. `tlv_stream`: `node_announcement_tlvs`
+2. types:
+   1. type: 1 (`option_will_fund`)
+   2. data:
+       * [`...*16*byte`:`lease_rates`]
+
+1. subtype: `lease_rate`
```

Comment:
An open question here is whether this should also include a `min_amount_sat` and `max_amount_sat`? This is especially useful to adapt the `funding_weight` to the seller's utxo set: if the buyer purchases a very large amount, this may require more inputs in the seller side, so the seller side may want to use a higher `funding_weight`.

@niftynei what do you think?

Reply:
I think @t-bast means "liquidity sellers to shape their utxo set".

ie. a seller might maintain a set of UTXOs which can effectively fund the amount(s) they advertise at different expected fee rates without needing to add extra inputs and producing no added change outputs.

Reply:
As discussed during the spec meeting, I have detailed that change in a follow-up commit here: https://github.com/t-bast/bolts/pull/1

Let me know what you think @rustyrussell @niftynei!